### ISSUE TITLE
Support job depencencies

### ISSUE THREAD

**Issue by mllg** [mllg/batchtools#190]
https://hpc.nih.gov/docs/job_dependencies.html

**Comment by jgrn307** [comment]
I think this would be great.  You could create/submit the jobs, and then when all of the jobs are completed have a "merge" job trigger without having the "submitter" job idling.

**Comment by wlandau** [comment]
Yes, thanks @mllg! This could make [`drake`](https://github.com/ropensci/drake) a lot more powerful, especially if [`future.batchtools`](https://github.com/HenrikBengtsson/future.batchtools) adopts the functionality too. cc @HenrikBengtsson

**Comment by jgrn307** [comment]
By the way, I'm happy to help test this out on a SLURM machine.  It seems a good first step would be to focus on batchtools combine-type functions.

**Comment by jgrn307** [comment]
Small correction with the above link -- when submitting the job via slurm, you will want to sbatch using the --parsable flag, e.g.:
sbatch --parsable [blah blah]
This will return ONLY the jobid and the cluster name separated by a semicolon (the jobid is what you will need for job dependencies in SLURM).  Without it, different slurm systems may output different text.  We tried the NIH link above initially and it didn't work due to the lack of the --parsable, we had it return something like "Your job ID is [jobid]", the text of which may differ from system to system.

See: https://slurm.schedmd.com/sbatch.html
